examples/balance/app.ddlog

```diff
@@ -0,0 +1,12 @@
+yes (vote_id bigint).
+no  (vote_id bigint).
+
+whatever? (
+    candidate bigint
+).
+
+@weight(1)
+whatever(p) :- yes(_).
```

Comment:
Grounding would fail with that syntax... I think currently we require all relations in the inference rule head to be a variable relation.